docs/source/management/index.md

```diff
@@ -108,16 +112,104 @@ Available SRE configurations for context 'green':
 
 ```{code} shell
 $ dsh sre teardown YOUR_SRE_NAME
 ```
 
+::::{admonition} Tearing down an SRE deletes ALL data and resources
+Running `dsh sre teardown` will remove all resources deployed within the SRE.
+Ensure that any desired outputs have been egressed before deleting the SRE.
+Any data remaining on the SRE will be deleted.
+The user groups for the SRE on Microsoft Entra ID will also be deleted.
+::::
+
 - Run the following if you want to teardown the deployed SHM:
 
 ```{code} shell
 $ dsh shm teardown
 ```
+
+::::{admonition} Tearing down an SHM
+All SREs associated with the SHM must be torn down before the SHM can be torn down.
+Tearing down the SHM deletes any remotely stored SRE configurations.
+::::
+
+## Managing data ingress and egress
+
+### Data Ingress
+
+It is the data provider's responsibility to upload the data required by the safe haven.
+
+```{important}
+Any data ingress must be signed off by the {ref}`role_data_provider_representative`, {ref}`role_investigator` and referee (if applicable).
+```
+
+The following steps show how to generate a temporary write-only upload token that can be securely sent to the data provider, enabling them to upload the data:
+
+- In the Azure portal select `Subscriptions` then navigate to the subscription containing the relevant SHM
+- Search for the resource group: `shm-<SHM ID>-sre-<SRE ID>-rg`, then click through to the storage account called: `sh<first three letter of SHM ID>sre<first three letters of SRE ID>sensitivedata`
+- Click `Networking` under `Settings` and ensure that the data provider's IP address is one of those allowed under the `Firewall` header, then hit the save icon in the top left
+- From the `Overview` tab, click the link to `Data storage` and then `Containers` (in the middle of the page)
+- Click `ingress`
+- Click `Shared access tokens` under `Settings` and do the following:
+    - Under `Signing method`, select `User delegation key`
+    - Under `Permissions`, check these boxes:
+        - `Write`
+        - `List`
+    - Set a 24 hour time window in the `Start and expiry date/time` (or an appropriate length of time)
+    - Leave everything else as default and click `Generate SAS token and URL`
+    - Copy the `Blob SAS URL`
+
+      ```{image} ingress_token_write_only.png
+      :alt: write-only SAS token
+      :align: center
+      ```
+
+- Send the `Blob SAS URL` to the data provider through a secure channel (for example, you could use the [Egress secure email](https://www.egress.com/) service)
+- The data provider should now be able to upload data
+- You can validate successful data ingress by logging into a workspace in the SRE and checking the `/mnt/input` volume, where you should be able to view the data that the data provider has uploaded
+
+### Data egress
+
+```{important}
+Any data egress must be signed off by the {ref}`role_data_provider_representative`, {ref}`role_investigator` and referee (if applicable).
+```
```

Comment:
Again, this feels out of scope.

Reply:
Egress in general?

Reply:
The part I don't like is "must be signed off by ..." because that is imposing some set of IG processes. I don't think this documentation is the right place to enforce or suggest what that process is.

I do think it is worth having a prominent warning that outputs may be sensitive and so you need to take care.